frontend/encore/simple-example.rst

```diff
@@ -333,8 +335,7 @@ If you want to only compile a CSS file, that's possible via ``addStyleEntry()``:
     Encore
         // ...
 
-        .addStyleEntry('some_page', './assets/css/some_page.css')
-    ;
+        .addStyleEntry('some_page', './assets/css/some_page.css');
```

Comment:
Let's revert this change because in most examples the `Encore`  config has multiple lines and we always put the last `;` in a new line. Thanks!